### DISCUSSION TITLE
Detect New Content Added

### DISCUSSION THREAD

**Discussion by jsalgado10** [OrchardCMS/OrchardCore#8256] [Q&A] [answered]
Hello

Is there any way to detect when new content is added. I am working on a notification system that will email users when new content is added/updated. 

Thanks

**Comment by Skrypt** [comment, marked as answer]
Workflows --> created event.
Or you can code your own Handler based on IContentHandler.

**Reply by jsalgado10** [reply]
The IContentHandler is what i am looking for. This will allow me to monitor the content that is being added/updated and send notifications to the users or do something else.
